recipe/build.sh

```diff
@@ -8,6 +8,7 @@ export ARROW_HOME=$PREFIX
 export PARQUET_HOME=$PREFIX
 export PYARROW_BUNDLE_ARROW_CPP_HEADERS=0
 export SETUPTOOLS_SCM_PRETEND_VERSION=$PKG_VERSION
+export PYARROW_WITH_ACERO=1
```

Comment:
Should we enable Flight as you did on the C++ library too? `-DPYARROW_BUILD_FLIGHT=ON`?

Reply:
yes, it is there on line 17